index.html

```diff
@@ -0,0 +1,48 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books</title>
+  <!-- <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    /> -->
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body onload="startTime()">
```

Comment:
- [ ] This `startTime()` function is throwing console error on the browser. Kindly fix it! :pray: 

![Screenshot from 2021-09-13 22-12-10](https://user-images.githubusercontent.com/64894927/133119600-a2503043-45fd-4f8d-bba1-d39ea1b3d2a1.png)